wasm/leb128/read.go

```diff
@@ -0,0 +1,73 @@
+package leb128
+
+import (
+	"io"
+)
+
+func ReadVarUintSize(r io.Reader) (res uint32, size uint, err error) {
```

Comment:
please add some documentation about `ReadVarUintSize` (especially what is the meaning of `res` and `size`)

also, as we have one set of functions returning `uint32` or `int32`, and another set returning `uint64` or `int64`, it would perhaps make sense to rename this set of functions as `ReadVarUint32Size` and `ReadVarUint32`.